_posts/2017-09-12-repeatable-serverless-applications.markdown

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: "Repeatable Serverless Applications"
+date: 2017-09-11
+categories: serverless architecture docker aws
+author: Rick Foxcroft
+image: img/no_servers.png
+---
+
+Servers can be expensive right? We aren't the biggest fan of lengthy provisioning processes, managing, maintenance and the costly nature that comes with deploying applications on servers. This doesn't mean that we avoid them altogether. We still have a monolithic application that uses several ec2 instances for numerous environments. On one of our recent mob-programming sessions, we decided to tackle the topic of, "How can we turn off any lurking, superfluous and idle ec2 instances" or any other idle instances we may have forgotten to stop/terminate. 
+
+![serverless benefits]({{ site.url }}/img/about_serverless_architecture.jpg)
```

Comment:
Serverless removes the need for ops and management? Errr... no. Or if you think yes you have to explain why.  Also it doesn't remove the need for scaling and fault tolerance, it satisfies it by being scalable and fault tolerant. Maybe flip the graphic on its head and say serverless _is_ "scalable, fault tolerant" and simplifies the other things.